src/unrepl/printer.clj

```diff
@@ -34,18 +34,18 @@
       (not (identical? impl default)))))
 
 (def ^:private datafiable?
-  (if-some [Datafiable (some-> 'clojure.core.protocols/Datafiable resolve deref)]
-    #(or (get (meta %) 'clojure.core.protocols/datafy) (really-satisfies? Datafiable %))
-    (constantly false)))
+  #(if-some [Datafiable (some-> 'clojure.core.protocols/Datafiable resolve deref)]
```

Comment:
The problem is the `deref`, could you just remove it from the `if-some` and put it in the anon fn on the line below?

Reply:
ok